README.md

```diff
@@ -60,6 +60,29 @@ Documentation on the `v2` structure can be found [here](https://github.com/ONSdi
 e.g."http://localhost:8000/quick-launch?schema_url=http://localhost:7777/1_0001.json&version=v1"
 ```
 
+### Commands for Formatting & Linting
+Install dependencies using `npm install` for formatting and linting
+
+| Command                 | Task                                                    |
+|-------------------------|---------------------------------------------------------|
+| `make format-static`    | Formats all static files (Javascipt and CSS)            |
+| `make format-templates` | Formats all HTML files and shows the changes to be made |
+| `make format-go`        | Formats all the Golang files                            |
+| `make format`           | Formats all files listed above                          |
+| `make lint-static`      | Lints all static files and reports any issues           |
+| `make lint-templates`   | Lints all HTML files and reports any issues             |
+| `make lint-go`          | Lints all Golang files using an external tool           |
+| `make lint`             | Lints all files listed above                            |
+
+**Note**: Before being able to run `lint-go`, you will need to install the external tool `golangci-lint`. The command to
```

Comment:
Only minor, but all the prerequisites should prob come before the table

Reply:
Makes sense - updated 👍